robbery.js

```diff
@@ -15,7 +22,24 @@ const isStar = true;
  * @returns {Object}
  */
 function getAppropriateMoment(schedule, duration, workingHours) {
-    console.info(schedule, duration, workingHours);
+    intersections = formatBankWorkingHours(workingHours);
+    const bankTimezone = workingHours.from.split('+')[1];
+
+    for (let member of Object.keys(schedule)) {
+        for (let daySchedule of schedule[member]) {
+            const tmp = daySchedule.from.replace(/[+:]/g, ' ').split(' ');
```

Comment:
`tmp` — плохое название переменной, потому что ни о чём не сообщает тому, кто читает код

Reply:
И зачем так сложно, если тебе нужно фактически
```suggestion
            const memberTimezone = Number(daySchedule.from.split('+')[1]);
```